account/tests.py

```diff
@@ -2,39 +2,62 @@
 import re
 from account.models import validate_gov_id, validate_phone, validate_pin_code
 from django.core.exceptions import ValidationError
+from account.factories import CustomUserFactory
+import django
 
+class TestCustomUserValidators(django.test.TestCase):
+    
+    def test_valid_phone_number(self):
+        user = CustomUserFactory()
```

Comment:
Given phone number here as well, do not rely on default value, if you are testing that value